app/mailers/deploy_mailer.rb

```diff
@@ -1,12 +1,9 @@
 # frozen_string_literal: true
 
 require_relative 'application_mailer'
-require 'inline_styles_mailer'
 
 class DeployMailer < ApplicationMailer
-  include InlineStylesMailer
   default from: 'deploy@invoca.com'
-  use_stylesheet 'table.scss'
```

Comment:
Just so that we don't loose the styling entirely, could you migrate the styles in [table.scss](https://github.com/Invoca/pre_deploy_checker/blob/master/app/assets/stylesheets/table.scss) to be applied as inline styles in the email template?

Reply:
Yes I have that in a TODO comment above. I just ran into another issue when running on rails 6.1 I think I'm about to get to the bottom of it